src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetLivePairing/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box } from 'grommet';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Video } from 'types/tracks';
+import { DashboardVideoLivePairing } from 'components/DashboardVideoLivePairing';
+import { DashboardVideoLiveWidgetTemplate } from 'components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetTemplate';
+
+const messages = defineMessages({
+  externalSourcesTitle: {
```

Comment:
```suggestion
  title: {
```
This is the id given to the message displayed in the widget's title in other widgets, and its also what you put in the id : 
`    id: 'components.DashboardVideoLiveControlPane.widgets.DashboardVideoLiveWidgetLiePairing.title'
`